tests/standalone_tests/algorithms/flower/test_df_filter.py

```diff
@@ -0,0 +1,353 @@
+import pandas as pd
```

Comment:
Since it will be very difficult to maintain and distinguish the test cases in the future you can use something similar to this:
https://github.com/madgik/exareme2/blob/1385732c8c57d8bddae03ec77f56ec52a7871dd4/tests/standalone_tests/controller/services/api/test_validate_algorithm_request.py#L519

It is actually naming each test case input so you know what you are testing.

Please try to add an explanatory name so that we can maintain it in the future.